lib/utils/testem-events.js

```diff
@@ -152,31 +161,56 @@ class TestemEvents {
     };
   }
 
+  /**
+   * Creates a file with targetJsonObject
+   *
+   * @param {*} fileName
+   * @param {*} targetJsonObject
+   * @param {*} option
+   */
+  _writeJsonToFile(fileName, targetJsonObject, option = {}) {
```

Comment:
This looks like a utility function, which does not depend on `this`, nor do we want/expect subclasses to override it. Let's move this off the prototype, and instead make it a top-level function in this module, as having it on the prototype is misleading (it suggests it depends on some internal instance state)